packages/dev/core/src/Materials/effect.ts

```diff
@@ -213,6 +213,10 @@ export class Effect implements IDisposable {
     private _processCodeAfterIncludes: ShaderCustomProcessingFunction | undefined = undefined;
     private _processFinalCode: Nullable<ShaderCustomProcessingFunction> = null;
 
+    public get shaderLanguage() {

```

Comment:
The doc is missing for the getter.

Reply:
Added doc and return type for `shaderLanguage` getter